code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -227,10 +227,14 @@
 
 /datum/action/xeno_action/evasion/can_use_action(silent = FALSE, override_flags)
 	. = ..()
+	var/mob/living/carbon/xenomorph/xeno_owner = owner
```

Comment:
very minor, but its better to put the var after the evade active check.
p.s. why the fuck doesn't the parent xeno_action just set a xeno_owner? Its used by fucking everything.

Reply:
BLeh owner is just a flat `mob`